tools/spec-gen-sdk/src/automation/reportStatus.ts

```diff
@@ -103,7 +106,7 @@ export const generateReport = (context: WorkflowContext) => {
     setPipelineVariables(context, executionReport);
   }
 
-  if (context.status === 'failed') {
+  if (['notEnabled', 'failed'].includes(context.status)) {
```

Comment:
Don't change this. We only fail the result when the generation fails.

Reply:
revert it